tests/test_pointing.py

```diff
@@ -0,0 +1,199 @@
+"Tests for pointing.py"
+
+# These tests only use 1 polarization axis
```

Comment:
I would move this comment to above the line `POLS = ["h"]`, otherwise you are going to forget about it...

You could also add "XXX" or "TODO" in front as a reminder that this could change in future.